src/Util/Metadata.php

```diff
@@ -19,14 +19,39 @@ class Metadata
      */
     protected $twig;
 
+    /**
+     * The env variable for GCMD Keyword version.
+     *
+     * @param gcmdVersion
+     */
+    protected $gcmdVersion;
```

Comment:
The @param is really needed, they is not type defined anyway.
Also I think it can be a private. Alternatively this can just be defined right in the constructor.

No action is needed, just for next time.